pinot-core/src/main/java/com/linkedin/pinot/core/data/extractors/PinotDataType.java

```diff
@@ -86,6 +87,17 @@ public Byte convert(Object value, PinotDataType sourceType) {
     }
   },
 
+  BYTES {
```

Comment:
Need to add toBytes()?

Reply:
I punted on that consciously, as unsure if we want to do that. If need be, can be done in following PR's.

Reply:
Done.